### PR TITLE
Pick flavours properly

### PR DESCRIPTION
**Pull request by mfalesni**
The flavours are named the same in various providers. I was not checking for them before.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3b8ef01596184e42c340e25325079704ffa1587d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/cloud/test_provisioning.py [hunk 1]
```diff
@@ -103,7 +103,14 @@ def test_provision_from_template_using_rest(
         else provisioning['instance_type'])
     flavours = rest_api.collections.flavors.find_by(name=instance_type)
     assert len(flavours) > 0
-    flavour_id = flavours[0].id
+    # TODO: Multi search when it works
+    for flavour in flavours:
```
Comment (seandst) [inline cfme/tests/cloud/test_provisioning.py]: The rest API uses the 'flavor' spelling, so while you're in here I'd love to see these all get fixed to be consistent with the thing we're testing.

Comment (psav) [inline cfme/tests/cloud/test_provisioning.py]: but the gui uses flavour......go figure :)

**Comment by dajoRH** [comment]
Lint report for commit ff481ba34a8969610d07ff02c1426483c9e811e6:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**